src/renderer.js

```diff
@@ -148,10 +148,21 @@ const Papa = require('papaparse');
             const decodedPassword = decodeLoginData(login.encryptedPassword);
             const username = decrypt(decodedUsername.data, decodedUsername.iv, key);
             const password = decrypt(decodedPassword.data, decodedPassword.iv, key);
+
+            // add timestamp infos
+            const timeCreated = Math.floor(login.timeCreated / 1000);
+            const timeLastUsed = Math.floor(login.timeLastUsed / 1000);
+            const timePasswordChanged = Math.floor(login.timePasswordChanged / 1000);
+            const timesUsed = login.timesUsed;
+
             logins.push({
                 hostname: login.hostname,
                 username: username.data,
-                password: password.data
+                password: password.data,
```

Comment:
Let's call the object properties the same thing that the file calls them. `timePasswordChanged`, `timesUsed`, etc.